src/controllers/AuthController.js

```diff
@@ -183,3 +183,13 @@ export const resetPassword = catchAsync(async (req, res, next) => {
   // Send success response
   res.status(200).json({ message: 'Password has been successfully reset' });
 });
+
+// Attempt to find a user in the database using the user ID.
+export const currentUser = catchAsync(async (req, res) => {
```

Comment:
`catchAsync` seems not to work without the `next` parameter, so you must add it after `res` to make the function work.